assets/css/app.css

```diff
@@ -2,6 +2,15 @@
 @import "tailwindcss/components";
 @import "tailwindcss/utilities";
 
+.full-width {
```

Comment:
Is it possible to achieve this with only tailwind classes? Just thinking about long term

Reply:
Challenge accepted :muscle: